### PR TITLE
Feature/enhancements ayesha 2

### PR DESCRIPTION
**Pull request by ayeshamk**

### CODE REVIEW

**Comment by ayeshamk** [comment]
@aih there are some conflicts. We both worked on the "no data" issues. Feel free to pick whichever you'd like to keep.

File: server_py/flatgov/bills/templatetags/bill_filters.py [hunk 1]
```diff
@@ -30,7 +30,7 @@ def billnumber_display(billnumber: str):
         number = billMatchGroups.get('number', '')
         stage_fmt = stagesFormat.get(stage.upper(), stage)
         if congress != constants.CURRENT_CONGRESS:
-           billnumber_fmt = f' {stage_fmt} {number} ({congress})'
+           billnumber_fmt = f' {stage_fmt} {number}'
```
Comment (aih) [inline server_py/flatgov/bills/templatetags/bill_filters.py:33]: This code supposed to put the congress in parentheses if it is not the current congress. With the change, we will never get the ({congress}) parenthetical.

File: server_py/flatgov/templates/bills/detail.html [hunk 2]
```diff
@@ -297,13 +301,48 @@ <h4>Sponsors of Related Measures</h4>
         <li class="nav-item" role="presentation">
             <a class="nav-link active" id="all-congress-sponsor-tab" data-toggle="tab" href="#all-congress-sponsor" role="tab" aria-controls="all-congress-sponsor" aria-selected="true">All Congresses</a>
         </li>
+        <li class="nav-item" role="presentation">
```
Comment (aih) [inline server_py/flatgov/templates/bills/detail.html:304]: "This Congresses" should be "Current Congress"; also, I'm not sure why it is being shown in a tab after "All Congresses". I don't see this reordering in Daniel's feedback.

**Review by aih — CHANGES_REQUESTED** [review]

**Comment by aih** [comment]
> @aih there are some conflicts. We both worked on the "no data" issues. Feel free to pick whichever you'd like to keep.

I've reverted my PR merge on the "no data" issues.

**Comment by aih** [comment]
We’re close on this PR. I don’t quite understand the ‘This Congress’ tab: 1. We have ‘Current Congress’ on the related bills tab. I think we should go with that, unless Daniel has asked otherwise. 2. The tab should go first, also unless Daniel asked otherwise.

**Comment by ayeshamk** [comment]
> We’re close on this PR. I don’t quite understand the ‘This Congress’ tab: 1. We have ‘Current Congress’ on the related bills tab. I think we should go with that, unless Daniel has asked otherwise. 2. The tab should go first, also unless Daniel asked otherwise.

He asked for a "this congress" tab. You can take a look at the Google Doc. Let me know what you'd like any changes.

**Comment by ayeshamk** [comment]
@aih done

**Review by aih — APPROVED** [review]

**Pull request merged**